appf.adoc

```diff
@@ -457,21 +407,19 @@ Expressed in the unit of the coordinate variable identified by the standard name
 If **`false_easting`** is not provided it is assumed to be 0.
 The formula to convert from the coordinate value as written in the **`projection_x_coordinate`** (xf) to a value (x0) used in a transformation without **`false_easting`**, i.e. **`false_easting`**= 0, is: x0 = xf -**`false_easting`**
 
-
-
 | **`false_northing`** | N
 |  Applied to all ordinate values in the rectangular coordinates for a map projection in order to eliminate negative numbers.
 Expressed in the unit of the coordinate variable identified by the standard name **`projection_y_coordinate`**.
 If **`false_northing`** is not provided it is assumed to be 0.
 The formula to convert from the coordinate value as written in the **`projection_y_coordinate`** (yf) to a value (y0) used in a transformation without **`false_northing`**, i.e.
 **`false_northing`**= 0, is: y0 = yf -**`false_northing`**
 
-| **`fixed_angle_axis `** | S
-| Indicates the axis on which the view is fixed in a hypothetical gimbal view model of the Earth, as used in the geostationary grid mapping.
-It corresponds to the inner-gimbal axis of the gimbal view model, whose axis of rotation moves about the outer-gimbal axis.
-This value can adopt two values, "x" or "y", corresponding with the Earth's E-W or N-S axis, respectively.
-The counterpart to this attribute is `sweep_angle_axis`.
-If set to "x", `sweep_angle_axis` is "y", and vice versa.
+| **`fixed_angle_axis`** | S
+| Indicates the axis on which the view is fixed in a hypothetical gimbal view model of the Earth, as used in the geostationary grid mapping. 
+It corresponds to the inner-gimbal axis of the gimbal view model, whose axis of rotation moves about the outer-gimbal axis. 
+This value can adopt two values, "x" or "y", corresponding with the Earth's E-W or N-S axis, respectively. 
+The counterpart to this attribute is `sweep_angle_axis`. 
+If set to "x", `sweep_angle_axis` is "y", and vice versa. 
```

Comment:
It looks like some trailing whitespace has been introduced here. Ideally we can revert back to not having this, in line with the style guide.